pkg/flow/logging/logger.go

```diff
@@ -57,9 +64,6 @@ func New(w io.Writer, o Options) (*Logger, error) {
 		},
 	}
 
-	if err := l.Update(o); err != nil {
-		return nil, err
-	}
```

Comment:
There is a problem with removing this (my bad I was the one suggesting it). It ignores the provided options and as a result it seems to create some obscure bugs in tests (the logger is used in many tests).
One way to fix it would be to keep this constructor with the update and to create a new constructor without the update call and without options. Something like "NewDeffered"? This would be used in the cmd_run.go and all the tests that depends on the loader would use the other constructor.